### PR TITLE
Docs fix: Fixed type on pc.Collision.asset as it can be a number or pc.Asset

### PR DESCRIPTION
**Pull request by yaustar**
Related to: https://github.com/playcanvas/engine/issues/2744

I confirm I have signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

**Comment by yaustar** [comment]
We could change the code in system to accept an pc.Asset type though

File: src/framework/components/collision/component.js [hunk 1]
```diff
@@ -46,8 +46,7 @@ import { Component } from '../component.js';
  * collision volume's length is aligned. 0 for X, 1 for Y and 2 for Z. Defaults to 1 (Y-axis).
  * @property {number} height The total height of the capsule, cylinder or cone-shaped collision
  * volume from tip to tip. Defaults to 2.
- * @property {Asset} asset The asset for the model of the mesh collision volume - can also be
- * an asset id. Defaults to null.
+ * @property {Asset} number The asset id for the model of the mesh collision volume. Defaults to null.
```
Comment (willeastcott) [inline src/framework/components/collision/component.js:49]: Well, first thing is that I think you meant:
```javascript
 * @property {number} asset The asset id for the model of the mesh collision volume. Defaults to null.
```
But as it happens, I think the code should be changed to take an asset. I believe that's more consistent with the model component:

https://github.com/playcanvas/engine/blob/master/src/framework/components/model/component.js#L549-L551

Comment (vkalpias) [reply src/framework/components/collision/component.js:49]: Usually components check if the value is a pc.Asset or an id and accept both.

**Comment by yaustar** [comment]
Updated PR to allow user of pc.Asset  as well as an id.

**Comment by yaustar** [comment]
Turns out the component already handles asset as an pc.Asset or number but both examples of this 'bug' are wrong in a different way (trigger script not added, wrong rotation).

Just a simple doc fix now 😅

**Review by willeastcott — APPROVED** [review]

**Pull request merged**